span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/SpanFilter.java

```diff
@@ -88,14 +119,23 @@ public SpanFilter(Config config) {
    * the span should be dropped, false otherwise.
    */
   public boolean shouldDropSpan(
-      JaegerSpanInternalModel.Span span, Map<String, JaegerSpanInternalModel.KeyValue> tags) {
+      JaegerSpanInternalModel.Span span,
+      Map<String, JaegerSpanInternalModel.KeyValue> tags,
+      Map<String, JaegerSpanInternalModel.KeyValue> processTags) {
     if (anyCriteriaMatch(tags, spanDropCriterion)) {
       if (DROPPED_SPANS_RATE_LIMITER.tryAcquire()) {
         LOG.info("Dropping span: [{}] with drop criterion: [{}]", span, spanDropCriterion);
       }
       return true;
     }
 
+    if (anySpanDropFiltersMatch(spanDropFilters, tags, processTags)) {
+      if (DROPPED_SPANS_RATE_LIMITER.tryAcquire()) {
```

Comment:
nit: can you put this block inside a debugger enabled check?

Reply:
There is `to string` not any eval, should it still matter?

Reply:
Ah! I see b'cause of `tryAcquire`, can do this.